samples/ts/21.react-media-template/src/live-share-hooks/useLiveCanvas.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { InkingManager, LiveCanvas } from "@microsoft/live-share-canvas";
+import { useCallback, useEffect, useState } from "react";
+
+/**
+ *
+ * @param {LiveCanvas} liveCanvas Fluid LiveCanvas Data object
+ * @param {HTMLElement} hostingElement Hosting Element where LiveCanvas needs to be attached.
+ * @returns LiveCanvas callback helpers and inking manager.
+ */
+export const useLiveCanvas = (
+    liveCanvas?: LiveCanvas,
+    hostingElement?: HTMLElement
+) => {
+    const [inkingManager, setInkingManager] = useState<InkingManager>();
+    const [error, setError] = useState<any>(undefined);
```

Comment:
Should be typed. 
```suggestion
    const [error, setError] = useState<any>();
```
It's okay to instantiate with no value, but we shouldn't add undefined or null - TS will handle the typing as long as there's `useState<string>` or whatever